Classes/IndexQueue/RecordMonitor.php

```diff
@@ -45,13 +43,6 @@ class RecordMonitor extends AbstractDataHandlerListener
 {
 
     /**
-     * Solr TypoScript configuration
-     *
-     * @var TypoScriptConfiguration
-     */
-    protected $solrConfiguration;
```

Comment:
Timo and I agreed that I would be better to hand it over explicitly instead of assuming that it has been initialised properly. In that way it also forces the developer to ensure that it is a "proper" configuration that is parsed.